### DISCUSSION TITLE
Removing/replacing footer text

### DISCUSSION THREAD

**Discussion by pa20bm** [wangchucheng/hugo-eureka#82] [Q&A]
This feels like a dumb question, but I can't figure out how to remove the "Write anything you like here!" text in the footer. I don't see anything that would control that element of the page in the homepage folder or any of the yaml files. 

You can see it hanging out at the bottom of the page here: https://pa20bm.github.io/HugoSite/

On a similar note, in the process of getting Eureka installed and working, I somehow managed to get "Cvs" at the top of this page – https://pa20bm.github.io/HugoSite/cv/ – when obviously I'd like it to just say "My C.V.," but I can no longer find any config files that control that text or even include the "cvs" text in them when I run a search on all of the files on my computer.

Any help would be greatly appreciated.

**Comment by wangchucheng** [comment]
I've converted this to discussion as this is the right place to look for support.

Firstly, that is a blank widget example instead of a footer actually. So you can just delete the `blank.html` in homepage folder.

Secondly, you can set title param in `<your-path>/_index.md`. This is called branch bundles in Hugo. For more: https://gohugo.io/content-management/page-bundles/#branch-bundles

**Reply by pa20bm** [reply]
Ah, OK, somewhere along the line, I must have deleted the _index.md for that folder (or just never created one). Is it as simple as creating that file and just putting the title I want in front matter?

**Reply by wangchucheng** [reply]
Yes!

**Reply by pa20bm** [reply]
Worked like a charm. Thanks so much!

**Comment by wangchucheng** [comment]
Have fun with Eureka. ;-)


________________________________
发件人: Paul Anthony ***@***.***>
发送时间: Monday, March 29, 2021 9:46:38 PM
收件人: wangchucheng/hugo-eureka ***@***.***>
抄送: C. Wang ***@***.***>; State change ***@***.***>
主题: Re: [wangchucheng/hugo-eureka] Removing/replacing footer text (#82)


Worked like a charm. Thanks so much!

―
